public/app/features/manage-dashboards/components/PublicDashboardListTable.tsx

```diff
@@ -0,0 +1,95 @@
+import { css } from '@emotion/css';
+import React, { useState } from 'react';
+import useAsync from 'react-use/lib/useAsync';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { getBackendSrv } from '@grafana/runtime';
+import { Link, ButtonGroup, LinkButton, Icon, Tag, useStyles2 } from '@grafana/ui';
+import { getConfig } from 'app/core/config';
+
+export interface ListPublicDashboardResponse {
+  uid: string;
+  accessToken: string;
+  dashboardUid: string;
+  title: string;
+  isEnabled: boolean;
+}
+
+export const LIST_PUBLIC_DASHBOARD_URL = `/api/dashboards/public`;
+export const getPublicDashboards = async () => {
+  const resp: ListPublicDashboardResponse[] = await getBackendSrv().get(LIST_PUBLIC_DASHBOARD_URL);
+  return resp.sort((a, b) => Number(b.isEnabled) - Number(a.isEnabled));
```

Comment:
We should sort them by name too, like dashboards.

Why do we want to sort them by status? Do we think enabled ones should be at the top?
In the future we are planning to allow enabling/disabling from this view. That could change the arrangement after disabling a dashboard. Is that desirable?

Reply:
Yeah, we should put enabled at the top. Instead of disabling from the table, there is now a deep link to the config panel. When you go back, it will reload the table. Will revisit when we get a request for bulk disable.

Reply:
Also- moving sorting to the database layer instead of the frontend.